tests/integ/modin/resample/test_resample.py

```diff
@@ -46,6 +46,43 @@ def test_resample_with_varying_freq_and_interval(freq, interval, agg_func):
     )
 
 
+@freq
+@interval
+@agg_func
+def test_resample_timedelta_and_floats_with_varying_freq_and_interval(
+    freq, interval, agg_func
+):
+    rule = f"{interval}{freq}"
+    # One extra query to convert index to native pandas for dataframe constructor
+    with SqlCounter(query_count=1):
+        dfs = create_test_dfs(
+            {
+                "A": np.random.randn(15),
+                "B": native_pd.timedelta_range("1 day", periods=15),
+            },
+            index=native_pd.date_range("2020-01-01", periods=15, freq=f"1{freq}"),
+        )
+
+    def operation(df):
+        return getattr(df.resample(rule=rule, closed="left"), agg_func)()
+
+    if agg_func == "var":
+        with SqlCounter(query_count=0):
+            eval_snowpark_pandas_result(
+                *dfs,
+                operation,
+                expect_exception=True,
+                expect_exception_type=TypeError,
```

Comment:
Since `eval_snowpark_pandas_result` by default has `assert_exception_equal=True`, it will check that the pandas exception message matches the Snowpark pandas exception message. I think that's sufficient here.